spikeinterface/comparison/hybridspikesrecording.py

```diff
@@ -0,0 +1,45 @@
+from typing import List, Union
+import numpy as np
+from spikeinterface.core import BaseSorting, NumpySorting, WaveformExtractor, AddTemplatesRecording
+
+
+class HybridSpikesRecording(AddTemplatesRecording):
+
+	def __init__(self, wvf_extractor: WaveformExtractor, max_n_spikes: int = 1000,
+				 max_frac_spikes: float = 0.05, refractory_period: float = 1.5) -> None:
```

Comment:
What does `max_frac_spikes` do?

Reply:
If this value is 0.05, it means that at the end, the maximum number of injected spikes in the whole spike train cannot exceed 5% (1,000 injected spikes is reasonable for a unit with 300,000 spikes, but not for a unit with 500).